vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -74,6 +77,23 @@ void HandleExpiredPromotions(
   }
 }
 
+ledger::PromotionList GetPromotionsForUI(ledger::PromotionMap* promotions) {
```

Comment:
Curious, does this need to be a pointer instead of const ref?

Reply:
No, good pooint - fixed.